src/test/java/uk/gov/hmcts/reform/sendletter/services/date/holidays/BankHolidaysClientTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.sendletter.services.date.holidays;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.sendletter.services.date.holidays.response.Event;
+import uk.gov.hmcts.reform.sendletter.services.date.holidays.response.Holidays;
+
+import java.time.LocalDate;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.notFound;
+import static com.github.tomakehurst.wiremock.client.WireMock.okJson;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+public class BankHolidaysClientTest {
+
+    @Rule
+    public WireMockRule api = new WireMockRule();
+
+    private BankHolidaysClient client;
+
+    @Before
+    public void setUp() {
+        this.client = new BankHolidaysClient(new RestTemplate(), "http://localhost:8080");
+    }
+
+    @Test
+    public void should_fetch_holidays() {
+        // given
+        api.stubFor(get("/").willReturn(
+            okJson("{"
+                + "\"division\": \"england-and-wales\","
```

Comment:
Is there only one division in the response?

Reply:
yes. see https://www.gov.uk/bank-holidays/england-and-wales.json